src/snmalloc/mem/backend_concept.h

```diff
@@ -110,8 +109,7 @@ namespace snmalloc
   template<typename Globals>
   concept ConceptBackendGlobals =
```

Comment:
Should the Backend inner class and its methods be captured here? Especially regarding the interfaces that are needed by the other layers to work.

Reply:
Yes, I have normally been very lazy with Concepts, and waited for @nwf-msr to get annoyed and add them, but I will try to be better ;-)

Reply:
Done, probably not as well as Wes, but he can take a pass in a subsequent PR.